Sources/NIO/ChannelOption.swift

```diff
@@ -225,9 +245,21 @@ extension ChannelOptions {
 
 /// Provides `ChannelOption`s to be used with a `Channel`, `Bootstrap` or `ServerBootstrap`.
 public struct ChannelOptions {
+    #if !os(Windows)
+        @available(*, deprecated)
```

Comment:
Can we use `renamed:` to help point people at the new one?

Reply:
Yeah